### PR TITLE
setup.py.in: Add entry-points

### PR DESCRIPTION
**Pull request by Adrianzatreanu**

### CODE REVIEW

File: bears/setup.py.in [hunk 1]
```diff
@@ -23,6 +23,9 @@ if __name__ == "__main__":
               license='$LICENSE',
               packages=find_packages(exclude=["build.*"]),
               long_description="""$LONG_DESCRIPTION""",
+              entry_points={"$BEAR_NAME": [
+                  "coala_official_$BEAR_NAME = "
+                  "bears.upload.$BEAR_NAME.$BEAR_NAME.$BEAR_NAME.$BEAR_NAME"]},
```
Comment (sils1297) [inline bears/setup.py.in]: wellll can we change this to a working import path? ;)

File: bears/setup.py.in [hunk 2]
```diff
@@ -23,6 +23,9 @@ if __name__ == "__main__":
               license='$LICENSE',
               packages=find_packages(exclude=["build.*"]),
               long_description="""$LONG_DESCRIPTION""",
+              entry_points={"$BEAR_NAME": [
```
Comment (sils1297) [inline bears/setup.py.in]: nah this has to be `coalabears` for it to work, I don't know if it's documented anywhere

File: bears/setup.py.in [hunk 3]
```diff
@@ -23,6 +23,9 @@ if __name__ == "__main__":
...
+                  "coala_official_$BEAR_NAME = "
```
Comment (sils1297) [inline bears/setup.py.in]: I'd just use `$BEAR_NAME`

File: bears/setup.py.in (hunk 1 above)
Comment (Adrianzatreanu) [inline bears/setup.py.in]: why wouldnt this one work?

**Comment by sils1297** [comment]
ack https://github.com/coala-analyzer/coala-bears/pull/665/commits/87118c8913ddf139e210854bd2c9a16b52751bb4

**Comment by Adrianzatreanu** [comment]
reack 8ce4334

**Comment by Adrianzatreanu** [comment]
@rultor merge

**Pull request merged**